go/vt/vtgate/planbuilder/operators/subquery.go

```diff
@@ -118,3 +118,14 @@ func createSubqueryFromStatement(ctx *plancontext.PlanningContext, stmt sqlparse
 	}
 	return subq, nil
 }
+
+func (s *SubQuery) Description() ops.OpDescription {
+	return ops.OpDescription{
+		OperatorType: "SubQuery",
+	}
+}
+func (s *SubQueryInner) Description() ops.OpDescription {
+	return ops.OpDescription{
+		OperatorType: "SubQueryInner",
+	}
+}
```

Comment:
I believe more information can be put here

Reply:
yeah, you are right. I wanted to do the minimal thing that I need to continue with my work. let's add to this description piece by piece as we find need for it